manifests/init.pp

```diff
@@ -107,8 +106,14 @@
   $stop_jira = 'service jira stop && sleep 15',
 
   # Tomcat
-  $tomcatAddress = undef,
-  $tomcatPort = 8080,
+  $tomcatAddress      = undef,
+  $tomcatPort         = 8080,
+  $tomcatNativeSsl    = false,
+  $tomcatHttpsPort    = 8443,
+  $tomcatKeyAlias     = 'jira',
+  $tomcatKeystoreFile = '/home/jira/jira.jks',
+  $tomcatKeystorePass = 'changeit',
+  $tomcatKeystoreType = 'JKS',
```

Comment:
Not introduced by this PR, but the interface mixes CamelCase, allowercase and snake_case. I feel that we should move modules towards consistently using underscores in parameter names. (Needs not happen in this PR.)